nvm/tasks/per_user.yml

```diff
@@ -0,0 +1,44 @@
+---
+- become_user: '{{ nvm_user_config.user }}'
+  become: true
+  block:
+
+  - name: Check NVM version
+    command: 'bash -ic "nvm --version"'
+    failed_when: nvm_installed_version.rc != 0 and nvm_installed_version.rc != 127
```

Comment:
What's with the 127? Does that need parametrising?

Reply:
No idea ::/:

Reply:
127 is the bash return code for command not found. I remember how this works now. If you say only `nvm_installed_version.rc != 127` then 0 is treated as a fail. I'll add a comment to make things clearer.

Reply:
Added in #0fa5a9b2fdbf627277fda11792484b7e24f7d928